cfgov/v1/jinja2/v1/layouts/layout-2-1.html

```diff
@@ -13,26 +13,42 @@
 
 {% else %}
 
-{% extends 'v1/layouts/content-base.html' %}
+{% extends 'v1/layouts/base.html' %}
 
-{% block body_content scoped %}
-{{ super() }}
-{% if not page or (page and page.language != 'ar') %}
-<div class="content_wrapper">
-    <div class="content_main {% block content_main_modifiers -%}{%- endblock %}">
-        <section>
-            {% block content_main scoped -%}{%- endblock %}
-        </section>
-    </div>
-    <aside class="content_sidebar {% block content_sidebar_modifiers -%}{%- endblock %}">
-        {% block content_sidebar scoped -%}{%- endblock %}
-    </aside>
-</div>
-{% endif %}
-{% endblock %}
+{% block content scoped %}
+    <main class="content content__2-1 content__bleedbar"
+          id="main"
+          {%- if is_faq_page %}
+            itemscope="" itemtype="http://schema.org/FAQPage"
```

Comment:
Wrote a bit about this on mattermost, but I think we should pull this into `base.html` so any page layout can be schema'd. Agree with your point that this is probably better done as a QAPage as well.

Reply:
Moved in https://github.com/cfpb/consumerfinance.gov/pull/8076/commits/a82b26b994825925f5bf3e6d2691cd1936f7b94d